tensorflow_serving/model_servers/main.cc

```diff
@@ -423,6 +446,17 @@ int main(int argc, char** argv) {
                        "Tensorflow session. Auto-configured by default."
                        "Note that this option is ignored if "
                        "--platform_config_file is non-empty."),
+      tensorflow::Flag("server_key", &server_key_path,
```

Comment:
Instead of four flags for this feature, can you create a protocol buffer definition and do it the way we do for platform_config_file? Thanks.